kiosk/src/Components/PlayingGame.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext, useMemo } from "react";
+import { AppStateContext } from "../State/AppStateContext";
+import configData from "../config.json";
+import "../Kiosk.css";
+
+export default function PlayingGame() {
+    const { state: kiosk, dispatch } = useContext(AppStateContext);
+    const { launchedGameId: gameId } = kiosk;
+
+    const playUrl = useMemo(() => {
+        if (gameId) {
+            const playUrlBase = `${configData.PlayUrlRoot}?id=${gameId}&hideSimButtons=1&noFooter=1&single=1&fullscreen=1&autofocus=1`;
+            const playQueryParam = kiosk.builtGamesCache[gameId]
+                ? "&server=1"
+                : "&sendBuilt=1";
```

Comment:
I might be misunderstanding this, but would we want to change the order for this ternary operator? If there is a game in the `builtGamesCache`, we would want the `sendBuilt=1` flag, right?

Reply:
You might be correct! Code is unchanged from the original: https://github.com/microsoft/pxt-arcade/blob/0b1189cbc8abc015854cc940e63f6ae9dc192de4/kiosk/src/Models/Kiosk.ts#L343

Reply:
I'd prefer not to change it in this PR, but will file an issue to investigate.

Reply:
Oh, interesting.. Yeah, this might be a bug. If we don't see performance improvements for subsequent game launches, then the order is wrong. If we do see those load time improvements, then there's something that I'm reading incorrectly. Like you said, something to investigate.